test/test_mlj.jl

```diff
@@ -0,0 +1,34 @@
+using ACEfit
+using LinearAlgebra
+using MLJ
+using MLJScikitLearnInterface
+
+@info("Test MLJ interface on overdetermined system")
+Nobs = 10_000
+Nfeat = 100
+A = randn(Nobs, Nfeat) / sqrt(Nobs)
+y = randn(Nobs)
+P = Diagonal(1.0 .+ rand(Nfeat))
+
+
+@info(" ... MLJLinearModels LinearRegressor")
+LinearRegressor = @load LinearRegressor pkg=MLJLinearModels
+solver = LinearRegressor()
+solver = ACEfit.LSQR(damp = 0, atol = 1e-6)
```

Comment:
I believe this line should be deleted?

Reply:
MLJ has different back ends. `MLJLinearModels` and `MLJScikitLearnInterface` are the ones that have been implemented. The point here is that these two behave differently on the background, so we need to test them both. So, there is one test for `MLJScikitLearnInterface` and one for `MLJLinearModels`. So, removing the part you suggest would leave `MLJLinearModels` without a test.

We could put here some solver that would be actually used, but all `MLJLinearModels` have the same interface on the background, so I thought that testing with LSQR would be the fastest.

Reply:
Hm, maybe I don't understand what's happening then. But it looks like we set solver equal to LinearRegressor() and then immediately overwrite it with a non-MLJ LSQR in the next line. Is that wrong?

By contrast the `MLJScikitLearnInterface` test makes sense to me.

Reply:
Nope, you are correct and I am the one who cannot read... 

Fixed the test and added the needed fix for parameter extraction.